From d9b5d88e94b8be0100b0fc2ee0054c78fb0737be Mon Sep 17 00:00:00 2001
From: Jiajie Li <lijiajie11@huawei.com>
Date: Tue, 13 Oct 2020 09:36:24 +0800
Subject: [PATCH 23/89] config: configure out the running arch

To support multi-arch, we need to figure out which architecture we stands on.
So let's get arch type before running the compilation process.

Signed-off-by: Jiajie Li <lijiajie11@huawei.com>
Signed-off-by: Ying Fang <fangying1@huawei.com>
---
 src/config | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100755 src/config

diff --git a/src/config b/src/config
new file mode 100755
index 0000000..6c44cf2
--- /dev/null
+++ b/src/config
@@ -0,0 +1,2 @@
+#!/bin/bash
+uname -m > arch.desc
-- 
2.23.0

